source/direct_debit/index.html.md.erb

```diff
@@ -292,11 +299,23 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>success</code></td>
-    <td>The payment succeeded.</td>
+    <td>GoCardless has taken the payment from your user’s bank account.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>paidout</code></td>
+    <td>GoCardless has sent the money to your bank account.</td>
+    </tr>    
+    <tr>
+    <td nowrap><code>cancelled</code></td>
+    <td>The payment was cancelled, for example because your user asked their bank to cancel the mandate.</td>
     </tr>
     <tr>
     <td nowrap><code>failed</code></td>
-    <td>The payment failed.</td>
+    <td>The payment failed, for example because your user did not have enough money in their bank account.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>indemnityclaim</code></td>
+    <td>After the payment succeeded, your user asked their bank for a refund ('indemnity claim') under the Direct Debit Guarantee.</td>
```

Comment:
worried this could be a little confusing as there are refunds (give me some money back as i paid too much for this thing) and indemnity claims (i never authorised this DD, so it's fraudulent). 
services will be able to give refunds, so perhaps clarify that a refund wouldn't show as an indemnity claim?

Reply:
Good point - I've clarified this in https://github.com/alphagov/pay-tech-docs/pull/306/commits/7ad6f07c2dc6e261a7724b1f8e665c2180950b91.